sans/reduction.ipynb

```diff
@@ -236,12 +247,11 @@
     "    # Setup direct beam and normalise to monitor. I.e. adjust for efficiency of detector across the wavelengths.\n",
     "    direct_beam = load_rkh(filename=f'{path}/{direct_beam_file}')\n",
     "    del direct_beam.coords['detector_info']\n",
-    "    del direct_beam.coords['position']\n",
     "    to_bin_edges(direct_beam, 'wavelength')\n",
     "    direct_beam = rebin_distribution(direct_beam, 'wavelength', monitor.coords['wavelength'])\n",
     "    direct_beam.unit = sc.units.one # TODO conflicts with unit conv to Q, is Q1D2 ignoring this?\n",
     "    direct_beam = monitor * transmission * direct_beam\n",
-    "    del direct_beam.coords['position']\n",
+    "    del direct_beam.coords['position'] # TODO already in wavelength -> never using TofBeamline -> non-standard coords\n",
```

Comment:
How can we avoid this elegantly? Is it something to be fixed in the converter?

Reply:
To clarify: I am thinking of something like making positions coords only if the workspace contains time-of-flight. If it is another unit, positions would be attributes.